### PR TITLE
Update of template.html with conditional statement

### PR DESCRIPTION
**Pull request by DamianHusted**
TLDR; Added boolean operator to report template. This mitigates having a listed vulnerability with the "in" preposition, when no URL/URI is present.

When generating reports, if a reported vulnerability does not have a specific URL/URI, the report will generate according to the following syntax (regardless of whether the URL is present or not): 
`<Vulnerability name> in <URL>`

This often results in a report with vulnerability headings appearing incomplete like: WAF detected in 

This change updates template.html file with a conditional check, avoiding this issue.

Vulnerability headings without a URL omit the "in" preposition.

### CODE REVIEW

**Review by yogeshojha — APPROVED** [review]
Excellent!

**Review by yogeshojha — APPROVED** [review]
